packages/babel-plugin-transform-react-jsx/src/index.js

```diff
@@ -15,6 +15,26 @@ export default declare((api, options) => {
   const JSX_ANNOTATION_REGEX = /\*?\s*@jsx\s+([^\s]+)/;
   const JSX_FRAG_ANNOTATION_REGEX = /\*?\s*@jsxFrag\s+([^\s]+)/;
 
+  if ("useSpread" in options) {
+    throw new Error(
+      'transform-react-jsx: "useSpread" is always enabled in Babel 8, please remove it from the config.\n',
```

Comment:
Although I guess we could also change the message. Not sure what would be better, "Since the "useSpread" is enabled by default in Babel 8, you can remove it from your config" @existentialism

Reply:
"Enabled by default" implies that you can disable it to me, and "you can remove it from your config" doesn't convey that it _must_ be removed.